src/include/platform/ConfigurationManager.h

```diff
@@ -64,6 +64,7 @@ class ConfigurationManager
 
     enum
     {
+        kMaxDeviceNameLen               = 32,
```

Comment:
Does this need to match Dnssd::kKeyDeviceNameMaxLength?  If so, can we static_assert that somewhere?

Reply:
https://github.com/ajoletta-amzn/connectedhomeip/blob/enable_dynamic_android_friendly_device_name/src/lib/dnssd/TxtFields.h#L50C1-L52C49


Added in this location.